repl/pom.xml

```diff
@@ -58,6 +58,11 @@
       <artifactId>spark-sql_${scala.binary.version}</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-connect_${scala.binary.version}</artifactId>
```

Comment:
We don't need to support this dsl in Spar repl. We can remove this and also move the dsl to `connect/src/test`

Reply:
done!